src/postgres-data/buildData.ts

```diff
@@ -348,6 +357,7 @@ export default async function buildData(
     user,
     api
   )
+  const productionNotes = await getProductionNotes(projectID, manuscriptID, api)
```

Comment:
Do we really need to request them separately? Are they not coming with all the other modes and then separated into its own array? See getManuscriptData function for reference.

Reply:
`/load` endpoint will not return by default notes or comments, @hhusban can we make the `/load` endpoint return notes and comments with other models?